plugins/obs-filters/noise-suppress-filter.c

```diff
@@ -247,6 +252,14 @@ static void *nvafx_initialize(void *data)
 				       err);
 				goto failure;
 			}
+			if (cuCtxGetCurrent(&curr) != CUDA_SUCCESS)
+				goto failure;
+
+			if (curr != old) {
+
```

Comment:
Remove blank line.

Reply:
this is fixed. Note though that this is from a commit that should be dropped in favour of the other PR (which doesn't have the issues btw).

Reply:
i mean, the two audio filter commits are provided to enable the build to work, but at merge time, they should be discarded.

Reply:
I am confused by this. Does that mean that this should not be merged as-is? If so, this should be a Draft PR.

Reply:
I'm going to leave this conversation unresolved to block merging until it's ready.

Reply:
Resolving this conversation now that the Audio PR has been merged.